cfme/middleware/datasource.py

```diff
@@ -41,12 +41,12 @@ def _db_select_query(name=None, nativeid=None, server=None, provider=None):
 
 def _get_datasources_page(provider=None, server=None):
     if server:  # if server instance is provided navigate through server page
-        server.summary.reload()
+        server.load_details()
```

Comment:
this is also unnecessary change as you click in the summary item the next thing. So usag eofsummary reload is a good idea here.